README.md

```diff
@@ -122,6 +122,23 @@ To upload stuff to S3, you'll need credentials. Your **AWS credentials** can be
 - If we can't find these, we'll look for an `.aws/config` file in the local environment.
 - NOTE: If the bucket is public, no credentials are necessary.
 
+### Dropbox
+Dropbox storage handler need user access token. For this:
+- Create Dropbox account (if you don't have).
+- Go to https://dropbox.com/developers
+- Create new application with App Folder (full access is not recommended, because without directory defined in connstring will have destructive results).
```

Comment:
```suggestion
- Create a new application with App Folder (full access is not recommended, because without directory defined in the configuration it will have destructive results).
```